src/design/optForce/optForce.m

```diff
@@ -0,0 +1,1000 @@
+%                            set 1   | upregulation    downregulation
+%                            set 2   | upregulation    knockout
+%    outputFileName.xls:     Type: file
+%                            Description: file containing 11 columns.
+%                            C1: Number of invervetions (k)
+%                            C2: Set Number
+%                            C3: Identifiers for reactions in the force set
+%                            C4: Type of regulations for each reaction in
+%                            the force set
+%                            C5: min flux of each reaction in force set,
+%                            according to FVA
+%                            C6: max flux of each reaction in force set,
+%                            according to FVA
+%                            C7: achieved flux of each of the reactions in
+%                            the force set after applying the inverventions
+%                            C8: objetive function achieved by OptForce.gms
+%                            C9: Minimum flux fot target when applying the
+%                            interventions
+%                            C10: Maximum flux fot target when applying the
+%                            interventions
+%                            C11: Maximum growth rate when applying the
+%                            interventions.
+%                            In the rows, the user can see each of the
+%                            optForce sets found.
+%    outputFileName.txt:     Same as outputFileName.xls but in a .txt file,
+%                            separated by tabs.
+%
+% NOTE: 
+%    This function is based in the GAMS files written by Sridhar
+%    Ranganathan which were provided by the research group of Costas D.
+%    Maranas. For a detailed description of the optForce procedure, please
+%    see: Ranganathan S, Suthers PF, Maranas CD (2010) OptForce: An
+%    Optimization Procedure for Identifying All Genetic Manipulations
+%    Leading to Targeted Overproductions. PLOS Computational Biology 6(4):
+%    e1000744. https://doi.org/10.1371/journal.pcbi.1000744
+%
+% .. Author: - Sebasti�n Mendoza, May 30th 2017, Center for Mathematical Modeling, University of Chile, snmendoz@uc.cl
+
+if nargin < 1 || isempty(model) % inputs handling
+    error('OptForce: No model specified');
+else
+    if ~isfield(model,'S'), error('OptForce: Missing field S in model');  end
+    if ~isfield(model,'rxns'), error('OptForce: Missing field rxns in model');  end
+    if ~isfield(model,'mets'), error('OptForce: Missing field mets in model');  end
+    if ~isfield(model,'lb'), error('OptForce: Missing field lb in model');  end
+    if ~isfield(model,'ub'), error('OptForce: Missing field ub in model');  end
+    if ~isfield(model,'c'), error('OptForce: Missing field c in model'); end
+    if ~isfield(model,'b'), error('OptForce: Missing field b in model'); end
+end
```

Comment:
You could use the verifyModel function with the set of required fields as option:
verifyModel('requiredFields',{'S','rxns','mets','lb','ub','c','b'}). This would at the same time check, whether these fields (and any other fields) are consistent with the definitions.